test/onnx/test_models_quantized_onnxruntime.py

```diff
@@ -0,0 +1,98 @@
+# Owner(s): ["module: onnx"]
+
+import os
+import unittest
+
+import parameterized
+import PIL
+import test_onnx_common
+import torchvision
```

Comment:
Both `transforms` and `quantization` are common module names and could potentially conflict and cause confusion. Since there is not heavy usage (1 per test case), I'm leaving it as is. Otherwise we could consider `from x import y as z`.